pkg/drivers/kic/oci/network.go

```diff
@@ -33,14 +35,18 @@ import (
 func RoutableHostIPFromInside(ociBin string, containerName string) (net.IP, error) {
 	if ociBin == Docker {
 		if runtime.GOOS == "linux" {
-			return dockerGatewayIP(containerName)
+			_, gateway, err := dockerNetworkInspect(containerName)
+			if err != nil {
+				return gateway, errors.Wrap(err, "routable gateway")
```

Comment:
This error annotation should be something like "network inspect". The caller to `RoutableHostIPFromInside` will probably add something like `routable host`.